homeworks/natalia.kokovina_NataliKokovina/homework_3/function.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-shadow */
+
+// 1.1 Exponentiation
+
+function Exponentiation(testNumber) {
+    console.log(testNumber ** 2);
```

Comment:
the condition is not to use `**` exponential operators

Reply:
fix